doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2748,7 +2757,7 @@ NCSDK-18263: |NCS| samples might fail to boot on Thingy:53
 Bluetooth samples
 =================
 
-.. rst-class:: v3-0-2 v3-0-1 v3-0-0
```

Comment:
Please add a new issue:

.. rst-class:: v3-1-0

NCSDK-34821: Fast Pair samples do not build or run due to the IronSide SE migration
  The :ref:`fast_pair_input_device` and :ref:`fast_pair_locator_tag` samples are currently incompatible with the latest |NCS| transition to the IronSide SE solution - bundle 22.1.0 and higher for the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
  As a result, you cannot use these samples with that board target in any of the affected releases.

  **Affected platforms:** nRF54H20

  **Workaround:** Use a future |NCS| release that includes IronSide SE support in the Fast Pair samples or use the |NCS| v3.0.0 release with the SUIT SDSC bundle (version up to 0.9.6).